packages/extension-store-locator/src/setup-app.ts

```diff
@@ -21,33 +21,28 @@ import StoreLocatorPage from './pages/store-locator'
 
 class StoreLocatorExtension extends ApplicationExtension<Config> {
     DEFAULT_PATH = '/store-locator'
-    REQUIRED_CONFIG_FIELDS = [
-        'defaultCountry',
-        'defaultCountryCode',
-        'defaultDistance',
-        'defaultDistanceUnit',
-        'defaultPageSize',
-        'defaultPostalCode',
-        'supportedCountries'
-    ]
+    DEFAULT_RADIUS = 100
+    DEFAULT_RADIUS_UNIT = 'km'
+    DEFAULT_PAGE_SIZE = 10
```

Comment:
Any special reason not to have all these configurations in a defaultConfig object? 

** Based on conversation, this might be changed later as we define a common default config pattern